3rdparty/singleapp/CMakeLists.txt

```diff
@@ -10,5 +10,5 @@ set_property(TARGET singleapp PROPERTY AUTOMOC ON)
 target_include_directories(singleapp SYSTEM PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
 target_include_directories(singleapp SYSTEM PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include/singleapp)
 
-target_link_libraries(singleapp PUBLIC Qt5::Widgets Qt5::Network)
+target_link_libraries(singleapp PUBLIC Qt${QT_VERSION_MAJOR}::Widgets Qt${QT_VERSION_MAJOR}::Network)
```

Comment:
Can we create something like aliases for Qt libraries, so we define something like
`add_library(QtWidgets ALIAS Qt${QT_VERSION_MAJOR}::Widgets)` and use our aliases everywhere, so that ${QT_VERSION_MAJOR} is used only in single place?